lib/postoffice_web/router.ex

```diff
@@ -33,7 +33,7 @@ defmodule PostofficeWeb.Router do
 
     resources "/publishers", PublisherController, only: [:index, :new, :create, :edit, :update, :delete]
 
-    resources "/messages", MessageController, only: [:index, :show]
+    resources "/messages", MessageController, only: [:index]
```

Comment:
Why did you remove the `:show` path without removing the function that used to take the requests? 🤔

Reply:
i forget to remove the code! Thanks a lot 😁😄